chapter11/sxg/ex_11_2.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+use DateTime;
+
+if ( @ARGV == 1 ){
+	my $now = DateTime−>from_epoch( epoch => time );
+	
+    
+	my $time = DateTime->new(year => $ARGV[0], month => $ARGV[1],day > $ARGV[2]);
+	        
```

Comment:
This script looks incorrect - you are getting into the conditional if there is a single argument passed to the script, but calling <code>$ARGV[0], $ARGV[1], $ARGV[2]</code> inside it. Also, wouldn't this script fail with syntax check? <code> day > $ARGV[2] </code>